client/reader/sidebar/index.jsx

```diff
@@ -193,7 +193,7 @@ export class ReaderSidebar extends Component {
 					className={ ReaderSidebarHelper.itemLinkClass( '/read', path, {
 						'sidebar-streams__following': true,
 					} ) }
-					label={ recentLabelTranslationReady ? translate( 'Recent' ) : translate( 'Following' ) }
```

Comment:
You can change the line to just:
```js
label={ translate( 'Home' ) }
```

The code is showing "Recent" or "Following" conditionally, depending if it's already translated.

And remove all the code references to `recentLabelTranslationReady`, `hasTranslation` and `locale`.